docs/accessibility.md

```diff
@@ -0,0 +1,221 @@
+
+## Interactive elements
+
+When creating elements that have event listeners, prefer any component of the `reactstrap` interactive elements. Examples are: `Button`, `Input`, `DropdownToggle`. You can also choose a HTML `<a>` element.
+
+If you need to insert an event listener in a non-interactive element, such as a `span`, add also an `aria-role` of `button`, `link`, `checkbox`, or whatever seems closer to the functionality of the element.
+
+```html
+<span onClick="{someFunction}" aria-role="button">
+  Unconventional button
+</span>
+```
+
+### Dropdown menu
+
+There is a special case when you are creating a dropdown menu. First of all, try to follow [`reactstrap` structure](https://reactstrap.github.io/components/dropdowns/).
+
+Lastly, insert an additional tag `prop` to `DropdownItem` component.
+
+```jsx
+<DropdownItem tag="a"> Menu Item </DropdownItem>
+```
+
+## Forms, inputs and buttons
+
+Apart from inserting high color contrast combinations in buttons, also make sure it has a `:focus` style (usually a blue outline is enough). If you use `reactstrap`'s `Button` component, this is added by default, hence it is recommended.
+
+In case the content of the button is not a text (an icon or graphical element, for example), this element should have an `aria-label`, explaining what it does.
+
+```jsx
+<Button aria-label="Bookmark this" onClick={onClickFunction}>
+  <FontAwesomeIcon icon={icon} />
+</Button>
+```
+
+For input and checkbox, those are important to be labeled - placeholders are not enough for screen reader users. For example, when inserting an email input and a checkbox, using `reactstrap`, the structure should be:
+
+```jsx
+<Form>
+  <FormGroup>
+    <Label for="email">Email</Label>
+    <Input
+      type="email"
+      name="email"
+      id="email"
+      placeholder="Insert your email"
+    />
+  </FormGroup>
+
+  <FormGroup>
+    <Label check>
+      <Input type="checkbox" /> I agree with the terms
+    </Label>
+  </FormGroup>
+</Form>
+```
+
+If in doubt, please check the [`reactstrap` Form documentation](https://reactstrap.github.io/components/form/).
+
+## Specific elements
+
+### Progress Bar
+
+Across Perfherder, there are some Progress Bars, which are built using `reactstrap` components. There is still an [issue](https://github.com/reactstrap/reactstrap/issues/1681), however, when adding labels on each bar. While this is still not solved, it is recommended to add an `aria-label` in the wrapping component, explaining the values of the bar.
+
+An example from the Alerts view:
+
+```jsx
+<Progress
+  multi
+  aria-label={`Magnitude of difference based on lower is better. Metric: ${
+    regression ? 100 - magnitude : magnitude
+  } % regressed`}
+>
+  <div aria-hidden="true" className="progress w-100">
+    <Progress bar value={regression ? 100 - magnitude : magnitude} />
+    <Progress bar value={regression ? magnitude : 100 - magnitude} />
+  </div>
+</Progress>
```

Comment:
Since this is an example, I think it'd be better to simply the code (no ternaries, just add values) so it's easier to follow.